### PR TITLE
feat: display about page with iris-client information

### PR DESCRIPTION
**Pull request by bianucci**
Refs iris-connect/iris-backlog#95

### CODE REVIEW

File: iris-client-fe/src/views/about/about.view.vue [hunk 1]
```diff
@@ -1,5 +1,83 @@
...
+import config from "@/config";
+
+@Component
+export default class AboutView extends Vue {
+  version = config.appVersionId;
+  build = config.appBuildId;
+  localContact = config.localContactPerson;
+
+  // TODO verify mail + phone
+  mail = "iris@steiger-stiftung.de";
```
Comment (faversente) [inline iris-client-fe/src/views/about/about.view.vue:79]: Maybe also make configurable like `VUE_APP_LOCAL_CONTACT` ?
Same for phone.

Comment (bianucci) [reply iris-client-fe/src/views/about/about.view.vue]: mail + phone have been removed because invalid

**Review by faversente — APPROVED** [review]

File: iris-client-fe/src/views/about/about.view.vue [hunk 2]
```diff
@@ -1,5 +1,82 @@
...
-    <h1>This is an about page</h1>
+    <h1 class="mb-6">Über IRIS</h1>
+
+    <v-card class="mb-6">
+      <v-card-title>Installation</v-card-title>
+
+      <v-list-item two-line>
+        <v-list-item-content>
+          <v-list-item-title>Version</v-list-item-title>
+          <v-list-item-subtitle>{{ version }}</v-list-item-subtitle>
```
[the remark this answers is not in the document]
Comment (SWlionmint) [reply iris-client-fe/src/views/about/about.view.vue:11]: Last push has removed the issue.

**Review by SWlionmint — APPROVED** [review]
Looks good

**Review by faversente — DISMISSED** [review]

**Review by hey-johnnypark — APPROVED** [review]
Looks good.

**Pull request merged**